### DISCUSSION TITLE
Any plan for migration of Svelte-Query docs to main repo?

### DISCUSSION THREAD

**Discussion by v3ss0n** [TanStack/query#6430] [General]
Current documentation is incomplete and very minimal https://tanstack.com/query/latest/docs/svelte/overview  , and i am not sure https://sveltequery.vercel.app/overview docs are still compatible. 
Any plan to merge those do main repo if compatible ?

**Comment by TkDodo** [comment]
Please contribute to the docs if you feel they are incomplete. Have a look at how e.g. the vue docs are based on the react docs on most pages with slight adaptations
